docs/cookbook/importing_settings_from_a_bundle.md

```diff
@@ -4,79 +4,70 @@
+This page shows how to import configuration from a bundle in two ways: the manual way and the implicit way.
 
-## Solution
+## Importing settings manually
 
-This recipe will show you how to import configuration from a bundle the manual way and the implicit way.
-
-### The manual way
-
-This is the simplest way of doing and it has the advantage of being explicit. The idea is to use the `imports` statement in your main `ezplatform.yml`:
+Importing manually is the simpler of the two ways and has the advantage of being explicit.
+It relies on using the `imports` statement in your main `ezplatform.yml`:
 
 ``` yaml
-# app/config/ezplatform.yml
 imports:
-    # Let's import our template selection rules that reside in our custom bundle.
-    # MyCustomBundle is the actual bundle name
-    - {resource: "@AcmeTestBundle/Resources/config/templates_rules.yml"}
+    # Import the template selection rules that reside in your custom AcmeExampleBundle.
+    - {resource: "@AcmeExampleBundle/Resources/config/templates_rules.yml"}
  
 ezpublish:
     # ...
 ```
 
+The `templates_rules.yml` should then be placed in `Resources/config` in AcmeExampleBundle.
+The configuration tree from this file will be merged with the main one.
+
 ``` yaml
-# templates\_rules.yml, placed under Resources/config folder in AcmeTestBundle
-# Here I need to reproduce the right configuration tree.
-# It will be merged with the main one
 ezpublish:
     system:
-        my_siteaccess:
-            ezpage:
-                layouts:
-                    2ZonesLayout1:
-                        name: "2 zones (layout 1)"
-                        template: "AcmeTestBundle:zone:2zoneslayout1.html.twig"
-
+        site_group:
             content_view:
                 full:
-                    article_test:
-                        template: "AcmeTestBundle:full:article_test.html.twig"
+                    article:
+                        template: "AcmeExampleBundle:full:article.html.twig"
                         match:
-                            Id\Location: [144,149]
-                    another_test:
-                        template: "::another_test.html.twig"
+                            Identifier\ContentType: [article]
+                    special:
+                        template: "::special.html.twig"
                         match:
                             Id\Content: 142
-
-            block_view:
-                campaign:
-                    template: "AcmeTestBundle:block:campaign.html.twig"
-                    match:
-                        Type: "Campaign"
 ```
 
-!!! note
+!!! caution
 
-    During the merge process, if the imported configuration files contain entries that are already defined in the main configuration file, **they will override them**.
+    The imported configuration will override the main configuration files if both cover the same settings.
```

Comment:
Why does it work this way?

Reply:
Does it really matter for the user in this case? As a cookbook recipe  I believe this is supposed to be a practical instruction, without going into technical details on the why's.